src/Features/Core/Portable/EmbeddedLanguages/DateAndTime/EmbeddedCompletionContext.cs

```diff
@@ -16,12 +17,30 @@ internal partial class DateAndTimeEmbeddedCompletionProvider
     {
         private readonly struct EmbeddedCompletionContext
         {
+            private static DateTime s_exampleDateTime = DateTime.Parse("2009-06-15T13:45:30.1234567Z");
```

Comment:
Tiny nitpick: wouldn't it be better to make this local time, instead of UTC, by removing the `Z`? That way, the time someone sees wouldn't depend on their time zone, which means the time can be chosen to showcase the difference between `hh` and `HH`? It would also be more consistent.

Reply:
Seems reasonable :)